src/common/bsod.cpp

```diff
@@ -272,7 +273,15 @@ void draw_error_screen(const uint16_t error_code_short) {
 
         /// draw QR
         char qr_text[MAX_LEN_4QR + 1];
-        error_url_long(qr_text, sizeof(qr_text), error_code);
+
+        /// switch for sending UID of printer or not
+        bool qr_privacy = variant_get_ui8(eeprom_get_var(EEVAR_QR_PRIVACY));
```

Comment:
probably we shall call the vars anonymized_qr to make it consistent with the menu item

Reply:
`eeprom_get_var` calls FreeRTOS's `osSemaphoreWait`. Is it safe, considering FreeRTOS is not running at that point?

Reply:
That's a very good question, but now BSOD screen is up right after reset, so it's ok and everything is running

Reply:
@Chleba is the eeprom itself already initialized at this point?